ui_framework/components/collapse_button/collapse_button.js

```diff
@@ -0,0 +1,29 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+const DIRECTIONS = [
+  'down',
+  'up',
+  'left',
+  'right'
+];
+
+export const KuiCollapseButton = ({ className, direction, ...rest }) => {
+  const classes = classNames('kuiCollapseButton', className);
+  const childClasses = classNames('kuiIcon', `fa-chevron-circle-${direction}`);
```

Comment:
Can we create a special map here, so it's easier to grep for the specific class names we're using? This is similar to the pattern used in `button.js`.

```jsx
import React, {
  PropTypes,
} from 'react';
import classNames from 'classnames';

const DIRECTIONS = [
  'down',
  'up',
  'left',
  'right',
];

const directionToClassNameMap = {
  down: 'fa-chevron-circle-down',
  up: 'fa-chevron-circle-up',
  left: 'fa-chevron-circle-left',
  right: 'fa-chevron-circle-right',
};

export const KuiCollapseButton = ({ className, direction, ...rest }) => {
  const classes = classNames('kuiCollapseButton', className);
  const childClasses = classNames('kuiIcon', direction[direction]);

  return (<button
            type="button"
            className={classes}
            {...rest}
          >
            <div className={childClasses}/>
          </button>);
};

KuiCollapseButton.propTypes = {
  className: PropTypes.string,
  direction: PropTypes.oneOf(DIRECTIONS).isRequired
};
```